cmd/tdf-decrypt.go

```diff
@@ -18,12 +19,26 @@ var assertionVerification string
 const TDF_MAX_FILE_SIZE = int64(10 * 1024 * 1024 * 1024) // 10 GB
 
 func dev_tdfDecryptCmd(cmd *cobra.Command, args []string) {
-	c := cli.New(cmd, args, cli.WithPrintJson())
+	c := cli.New(cmd, args)
```

Comment:
I know you removed `cli.WithPrintJson()` for debugging purposes, but I think removal could corrupt the bytes that are output to stdout with miscellaneous logs that would prevent a valid TDF from being piped to a file. I think we should add it back here and handle the need for debug behavior when decrypt fails in a separate issue.